### PR TITLE
Fix etree imports for py3.9-readiness

### LINKED ISSUE #2002: xml.etree.cElementTree removed from Python 3.9
With newer versions of Python, `xml.etree.ElementTree` should be used directly.

### PR DESCRIPTION
**Pull request by caje731**
Fixes #2002

### CODE REVIEW

**Comment by caje731** [comment]
From the failing test in `tests/functional/retries/test_bucket.py`:

`+        mean = sum(distribution) / float(len(distribution))
+        # We can't really rely on any guarantees about evenly distributing
+        # thread acquisition(), e.g. must be with a 2 stddev range, but we
+        # can sanity check that our implementation isn't drastically
+        # starving a thread.  So we'll arbitrarily say that a thread
+        # can't have less than 30% of the mean allocations per thread.
+        self.assertTrue(not any(x < (0.3 * mean) for x in distribution))
`

Seems to be a probabilistic test introduced this January, and is not guaranteed to pass - in fact I see more than one other PR failing due to this unrelated test. IMO this test case warrants fixing too (through another PR?)

File: tests/functional/retries/test_bucket.py [hunk 1]
```diff
@@ -105,5 +105,5 @@ def test_stress_test_token_bucket(self):
         # thread acquisition(), e.g. must be with a 2 stddev range, but we
         # can sanity check that our implementation isn't drastically
         # starving a thread.  So we'll arbitrarily say that a thread
-        # can't have less than 30% of the mean allocations per thread.
-        self.assertTrue(not any(x < (0.3 * mean) for x in distribution))
+        # can't have less than 20% of the mean allocations per thread.
```
[the remark this answers is not in the document]
Comment (caje731) [reply tests/functional/retries/test_bucket.py]: My last push has reverted this change, but I'm not fully sure why you'd want to keep it flaky.

File: botocore/handlers.py [hunk 2]
```diff
@@ -104,6 +104,13 @@ def check_for_200_error(response, **kwargs):
 
 def _looks_like_special_case_error(http_response):
     if http_response.status_code == 200:
+<<<<<<< HEAD
```
Comment (tremble) [inline botocore/handlers.py:107]: Looks like an artefact from a bad merge / rebase

Comment (caje731) [reply botocore/handlers.py]: Indeed! Thanks for pointing out!
Feeling foolish for letting that happen. Anyhow, fixed - please review again, thanks!

File: botocore/compat.py [hunk 3]
```diff
@@ -146,8 +146,12 @@ def ensure_bytes(s, encoding='utf-8', errors='strict'):
 from collections import OrderedDict
 
 
-import xml.etree.cElementTree
-XMLParseError = xml.etree.cElementTree.ParseError
+try:
+    import xml.etree.cElementTree as ETree
+except ImportError:
+    # cElemenTree does not exist from Python3.9+
```
Comment (tremble) [inline botocore/compat.py:152]:
```suggestion
    # cElementTree does not exist from Python3.9+
```

**Review by tremble — COMMENTED** [review]
Another typo, but for what little it's worth otherwise looks sane to me

File: botocore/compat.py (hunk 3 above)
Comment (caje731) [reply botocore/compat.py]: Thanks, feels good to be reminded of all my drunk-commits lol. Fixed the typo.

**Pull request merged**